pwiz_tools/Skyline/Controls/Startup/StartPage.cs

```diff
@@ -434,6 +434,18 @@ private void PopulateTutorialPanel()
                         string.Empty
                     ),
                     Description = TutorialTextResources.HiResMetabolomics_Description
+                },
+                new TutorialActionBoxControl
+                {
+                Caption = TutorialTextResources.SmallMolLibraries_Caption,
+                Icon = TutorialImageResources.SmallMolLibraries_start,
+                EventAction = () => Tutorial(
+                    ActionTutorial.TutorialType.hi_res_metabolomics,

```

Comment:
All other tutorials seem to have their own type. So, I am guessing this is a copy-paste error where you forgot to give this new tutorial its own value in the enum. Though, I am not seeing how this value gets used. Seems vestigial and should maybe be removed entirely.

Reply:
Good catch, thanks - I'll see whether this is needed at all, as you suggest.

Reply:
Yeah, just cruft. I tore out all the ActionTutorial.TutorialType stuff.